### PR TITLE
Clean up ipvs proxy startup in local-up.sh

### PR DESCRIPTION
**Pull request by m1093782566**
**What this PR does / why we need it**:

Clean up ipvs proxy startup in local-up.sh

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

/sig network

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -743,16 +743,12 @@ clientConnection:
   kubeconfig: ${CERT_DIR}/kube-proxy.kubeconfig
 hostnameOverride: ${HOSTNAME_OVERRIDE}
 featureGates: ${FEATURE_GATES}
+mode: ${KUBEPROXY_MODE}
 EOF
     if [ "${KUBEPROXY_MODE}" == "ipvs" ]; then
-        cat <<EOF >> /tmp/kube-proxy.yaml
-mode: ipvs
 EOF
```
Comment (cblecker) [inline hack/local-up-cluster.sh]: I think this line can go too?

Comment (m1093782566) [reply hack/local-up-cluster.sh]: Yes. that's a nice finding :)

Comment (m1093782566) [reply hack/local-up-cluster.sh]: Fixed now.

**Comment by cblecker** [comment]
/approve no-issue
Leaving LGTM to @caseydavenport and @cmluciano :)

**Comment by caseydavenport** [comment]
Thanks!

/lgtm

**Pull request merged**